pages/pipelines/trigger_step.md.erb

```diff
@@ -127,7 +127,12 @@ _Optional_ `build` _attributes:_
     branch: "${BUILDKITE_BRANCH}"
 ```
 
-<div class="Docs__note">
-<p class="Docs__note__heading">Environment variable support</p>
-<p>Buildkite Agent 3.0 or above is required to use pipeline.yml environment variable substitution. See the <a href="/docs/agent/v2/cli-pipeline#environment-variable-substitution">pipeline command documentation</a> for more information.</p>
-</div>
```

Comment:
Do you think enough people are on the v3 agent to remove this note? 

Related thought: It would be good to have some note somewhere that tells people we assume you're on the latest stable agent, or something along those lines.